docs/user_roles.md

```diff
@@ -62,15 +62,28 @@ to the course:
 
 ... or you can visit the enroll URL (visible to the instructor) using the student account.
 
+## Populating initial courses
+
+You can set up a few standard courses by running the following rake task:
+
+```
+rake dev:populate
+```
+
+Go to the **Explore** or **Find Programs** page to see the created campaigns and courses.
+
 ## Populating a course with data
 
 The data for Articles, Activity, Uploads and so on for a course comes from activity
 on Wikipedia by the participants. You can add active users on Wikipedia to pull in
 arbitrary activity data.
 
-1. Go to Wikipedia and find a recently-active editors, for example by picking some usernames from https://en.wikipedia.org/wiki/Special:RecentChanges
-2. For populating the Uploads, find users who have recently uploaded files by picking some usernames from https://en.wikipedia.org/wiki/Special:Log/upload
-3. As an instructor or admin, go the Students tab, click Participation, and add them.
+To reproduce a working course with active editors, complete the following:
+
+1. Go to Wikipedia and find the usernames of [recently active editors](https://en.wikipedia.org/wiki/Special:RecentChanges).
+1. For populating the Uploads, find users [who have recently uploaded files](https://en.wikipedia.org/wiki/Special:Log/upload).
```

Comment:
Does this numbering render automagically to a numbered list, or is the repeated `1` a mistake?

Reply:
It's [auto-magical!](https://github.com/WikiEducationFoundation/WikiEduDashboard/blob/6f6a639ea8b3da65f3b2c1d1d5ebbd2b9fc52cf1/docs/user_roles.md)